chains/ethereum/config.go

```diff
@@ -50,5 +56,21 @@ func ParseChainConfig(chainCfg *core.ChainConfig) *Config {
 		config.emitter = common.HexToAddress(emitter)
 	}
 
+	if gasPrice, ok := chainCfg.Opts["gasPrice"]; ok {
+		matched, err := regexp.MatchString("^[0-9]+$", gasPrice)
+		if matched && err != nil {
+			price64, _ := strconv.ParseInt(gasPrice, 10, 64)
+			config.gasPrice = big.NewInt(price64)
+		}
+	}
```

Comment:
This is unsafe and overcomplicated. Please use this https://golang.org/pkg/math/big/#Int.SetString

Reply:
Addressed